lib/elixir/pages/getting-started/keywords-and-maps.md

```diff
@@ -27,22 +29,46 @@ iex> String.split("1  2  3", " ", [trim: true])
 ["1", "2", "3"]
 ```
 
-`[trim: true]` is a keyword list. Furthermore, when a keyword list is the last argument of a function, we can skip the brackets and write:
+We can also use options to limit the splitting algorithm to a maximum number of parts, as shown next:
 
 ```elixir
-iex> String.split("1  2  3", " ", trim: true)
-["1", "2", "3"]
+iex> String.split("1  2  3", " ", [trim: true, parts: 2])
+["1", "2  3"]
+```
+
+`[trim: true]` and `[trim: true, parts: 2]` are keyword lists. When a keyword list is the last argument of a function, we can skip the brackets and write:
+
+```elixir
+iex> String.split("1  2  3", " ", trim: true, parts: 2)
+["1", "2  3"]
 ```
 
 As shown in the example above, keyword lists are mostly used as optional arguments to functions.
 
 As the name implies, keyword lists are simply lists. In particular, they are lists consisting of 2-item tuples where the first element (the key) is an atom and the second element can be any value. Both representations are the same:
 
 ```elixir
-iex> [{:trim, true}] == [trim: true]
+iex> [{:trim, true}, {:parts, 2}] == [trim: true, parts: 2]
 true
 ```
 
+Keyword lists are important because they have three special characteristics:
+
+  * Keys must be atoms.
+  * Keys are ordered, as specified by the developer.
+  * Keys can be given more than once.
+
+For example, we use the fact keys can be repeated when [importing functions](../getting-started/alias-require-and-import.md) in Elixir:
```

Comment:
I replaced the Ecto example by `import`, which is part of Elixir, and may be less confusing. WDYT?

Reply:
this looks clearer to me and i believe a better choice!